tests/test_vertrag.py

```diff
@@ -0,0 +1,184 @@
+from datetime import datetime, date
+import pytest
+from bo4e.bo.geschaeftspartner import Geschaeftspartner
+from bo4e.bo.vertrag import Vertrag, VertragSchema
+from bo4e.com.unterschrift import Unterschrift
+from bo4e.com.vertragskonditionen import Vertragskonditionen
+from bo4e.com.vertragsteil import Vertragsteil
+from bo4e.enum.botyp import BoTyp
+from bo4e.enum.sparte import Sparte
+from bo4e.enum.vertragsstatus import Vertragsstatus
+from bo4e.enum.anrede import Anrede
+from bo4e.enum.kontaktart import Kontaktart
+from bo4e.enum.geschaeftspartnerrolle import Geschaeftspartnerrolle
+from bo4e.com.adresse import Adresse
+from bo4e.enum.vertragsart import Vertragsart
+
+
+class TestVertrag:
+    _vertragsnummer = "1234567891011"
+    _vertragsart = Vertragsart.BILANZIERUNGSVERTRAG
+    _vertragsstatus = Vertragsstatus.AKTIV
+    _sparte = Sparte.STROM
+    _vertragsbeginn = datetime(2021, 4, 30, 13, 45)
+    _vertragsende = datetime(2021, 6, 5, 16, 30)
```

Comment:
```suggestion
    _vertragsbeginn = datetime(2021, 4, 30, 13, 45).replace(tzinfo=timezone.utc)
    _vertragsende = datetime(2021, 6, 5, 16, 30).replace(tzinfo=timezone.utc)
```
mit `from datetime import timezone`